cirq/ops/clifford_gate_test.py

```diff
@@ -359,12 +359,8 @@ def test_y_rotation(gate, trans_y):
     (cirq.SingleQubitCliffordGate.Z_nsqrt, cirq.Z ** -0.5)))
 def test_decompose(gate, gate_equiv):
     q0 = cirq.NamedQubit('q0')
-    mat = cirq.Circuit.from_ops(
-                    gate(q0),
-                ).to_unitary_matrix()
-    mat_check = cirq.Circuit.from_ops(
-                    gate_equiv(q0),
-                ).to_unitary_matrix()
+    mat = cirq.Circuit.from_ops(gate(q0),).unitary()
+    mat_check = cirq.Circuit.from_ops(gate_equiv(q0),).unitary()
```

Comment:
Nit: Unneeded commas on the same line as the end parenthesis.

Reply:
Removed.